Extensions/Video/videoruntimeobject.js

```diff
@@ -45,6 +59,7 @@ gdjs.VideoRuntimeObject.prototype.getRendererObject = function() {
 /**
  * Initialize the extra parameters that could be set for an instance.
  * @private
+ * @param {{customSize: {width: number, height: number}}} initialInstanceData The initial instance data
```

Comment:
We should create an InitialInstanceData type, but I'm not sure where to do that. Maybe in `runtimeobject.js`.

Reply:
Don't objects have different ones?

Reply:
They can, so like objects are created with `ObjectData` and some specific fields according to the object, their instances can be initialized with `InitialInstanceData`, which always contain things like the layer, customSize and a bunch of other properties. And some objects can have extra information stored there (like sprites)

Reply:
Ok will do In another PR. Should I create an issue to remember?

Reply:
We can either go for a:
* single issue for all typings stuff
* or comments/checklists (that I can add if you comment) to: https://trello.com/c/qy3QQ7qD/350-progressively-add-typescript-type-checking-to-the-game-engine

As you prefer!